CONTRIBUTING.md

```diff
@@ -49,15 +49,15 @@ To get started, run `bin/setup`
 
 After setting up, you can run the application using [foreman]:
 
-    % foreman start
+    foreman start
 
 If you don't have `foreman`, see [Foreman's install instructions][foreman]. It
 is [purposefully excluded from the project's `Gemfile`][exclude].
 
 [foreman]: https://github.com/ddollar/foreman
 [exclude]: https://github.com/ddollar/foreman/pull/437#issuecomment-41110407
 
-The application will run locally at `http://localhost:5000/`.
+The application will run locally at http://localhost:5000/.
```

Comment:
this seems conflicting with https://github.com/18F/dolores-landingham-bot/pull/166/files ?